### DISCUSSION TITLE
Only allow certain apps to access local network (local network split-tunneling)

### DISCUSSION THREAD

**Discussion by I-I-IT** [mullvad/mullvadvpn-app#9127] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

Currently, you can either block/allow ALL apps to access local network OR exclude certain apps from the VPN altogether.
I propose that you could only allow certain apps to access local network. For example, **I want my browser and document editing to access local network for printing, but I certainly do not want other closed source apps, such as Netflix, Spotify, etc to be able to access the local network, because this could potentially track my location.**

### Alternative solutions

You can always manually enable and disable local network sharing, at a great usability or time burden.

### Type of feature

- [X] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [ ] Easier to use
- [ ] Other

### Operating System

- [ ] Android
- [ ] iOS
- [X] Windows
- [ ] macOS
- [X] Linux

**Comment by hulthe** [comment]
Thank you for the suggestion, I agree that this would be nice to have.

I'll bring this up with the team :)

**Comment by theafterglow** [comment]
do we want the same thing?
i wish we could only let certain apps to use the vpn, the opposite of what we currently have.

**Comment by MrChocolatine** [comment]
> do we want the same thing? i wish we could only let certain apps to use the vpn, the opposite of what we currently have.

@chifuno it looks like what you are asking is #2808 .

**Comment by theafterglow** [comment]
> > do we want the same thing? i wish we could only let certain apps to use the vpn, the opposite of what we currently have.
> 
> @chifuno it looks like what you are asking is #2808 .

right and it doesn’t look like this feature will be added...

**Comment by I-I-IT** [comment]
@chifuno Mullvad always focused on privacy and censorship resistance. Users may want other feature, but it isn't their focus.
